### ISSUE TITLE
Rethink toctree_only=False behavior

### ISSUE THREAD

**Issue by brechtm** [sphinx-doc/sphinx#2717]
When `toctree_only` is disabled, the LaTeX builder places the `startdoc` contents in a chapter without a title. The title of this chapter is "Contents", as displayed in the page footer.

I'm thinking this could be improved upon, but I'm not sure how. Perhaps the startdoc contents could be rendered as a foreword or preface, or perhaps as a chapter of which the title can be configured (default to "Introduction")? I guess this could depend on the document template used (article or book).

**Comment by andreacassioli** [comment]
I agree this is an annoying issue. I think there should be an option to set that title if the section is not empty.

**Comment by jfbu** [comment]
The `Contents` in the page footer comes from the table of contents. As no new chapter has yet started the footer is not changed. One could imagine for example render the contents of the `startdoc` as an Abstract. But this would possibly break existing things.

Personally I find that sphinx-quickstart's way of putting a default line "Contents:" is confusing for latex target. latex writer puts the `\tableofcontents` command before anything else found in `startdoc`. Hence `Contents:` ends up on a page of its own after it. As I was disconcerted by that when I started using Sphinx I have gotten in to the habit to put contents in `startdoc` only for testing small rst snippets. For real documents I don't have any text contents there. I wonder what other people do :-)
